recipe/meta.yaml

```diff
@@ -1,21 +1,21 @@
 {% set name = "lightning" %}
-{% set version = "2.0.9.post0" %}
+{% set version = "2.3.3" %}
```

Comment:
Not sure if we needed 2.3.3 specifically, but just in case FYI, 2.4.0 is available and 2.5.0 is already in the works.

Reply:
In this case the upper-bound in the dependent package is `<2.4` so it's just outside the range. I appreciate the heads-up !